web/src/tableUtils.ts

```diff
@@ -0,0 +1,12 @@
+export function formatNumber(
```

Comment:
Could you add a docstring on this one? On the format
```suggestion
/**
 *
 * @param {string} param0
 * @returns
 */
export function formatNumber(
```
It is not obvious what the threshold or precision is, and what's their default values

Reply:
Good suggestion. Will do.